app/components/user_programme_course_bookings_with_asides_component/user_programme_course_bookings_with_asides_component.html.erb

```diff
@@ -0,0 +1,79 @@
+<%= render GovGridRowComponent.new(padding: {top: 0, bottom: 7}) do |row| %>
+  <%= row.with_column("two-thirds") do %>
+    <div class="user-programme-course-bookings-with-asides">
+      <div class="user-programme-course-bookings-with-asides__header">
+        <% if @programme.programme_activity_groupings.not_community.first.user_complete?(@current_user) %>
+          <div>
+            <%= image_pack_tag('media/images/icons/activity-complete-icon.svg') %>
+          </div>
+        <% end %>
+        <div>
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-0">Step one: CPD courses</h2>
+        </div>
+      </div>
+
+      <% if @courses[:in_progress].present? %>
```

Comment:
These next two loops are the same I believe?

If so could we find some way to combine then to reduce duplication

Reply:
Good spot, definitely not a dry approach here